static/sass/_pattern_card.scss

```diff
@@ -115,6 +115,10 @@
     display: flex;
     justify-content: center;
     min-height: 10rem;
+
+    img {
+      align-self: center;
+    }
```

Comment:
Please don't target element in patterns. I would prefer we used `u-vertically-center` on the link around the images instead of this.